src/paste-keyboard-shortcut-helper.ts

```diff
@@ -21,13 +23,22 @@ export function shouldSkipFormatting(el: HTMLElement): boolean {
   return shouldSkipFormattingState
 }
 
-export function installAround(el: HTMLElement, ...installCallbacks: Array<(el: HTMLElement) => void>): void {
+export function installAround(
+  el: HTMLElement,
+  installCallbacks: Array<(el: HTMLElement) => void>,
+  installCallbacksWithOptions: Array<(el: HTMLElement, optionConfig?: OptionConfig) => void>,
```

Comment:
rather than have two arrays here, why not just pass options to all the callbacks?

Reply:
I was hoping to avoid confusion given we export the individual installation functions, too, but the documentation update would be sufficient. Let me tackle this now

Reply:
Infact I am not even sure this is needed, I believe still keeping them to the same array and simplying defaulting to passing in config params in all cases works (tests are still passing).

<img width="317" alt="image" src="https://user-images.githubusercontent.com/13091188/191440005-2c34d9a3-4d41-4081-a659-55ff68c68fcc.png">

If my understanding is correct, it'll simply ignore the param